_lessons/python-scatter/02-vectorising.md

```diff
@@ -21,9 +21,9 @@ cd vect
 
 ## What is vectorisation
 
-Vectorisation in Python is a programming style where operations on a single piece of data, typically in a loop, are replaced by operations on entire arrays. Vectorisation can improve the performance of a code and can make the code more concise and easier to maintain.
+Vectorisation in Python is a programming style where operations on a single piece of data, typically in a loop, are replaced by operations on entire **arrays**. Vectorisation can improve the performance of a code and can make the code more concise and easier to maintain.
 
-In scripting languages, loops can be slow to execute because of the overhead of the interpreter, which may need to parse each expression, perform various input data checks and more. These overheads add up when expressions are repeated many times in a loop. Vectorisation avoids the issue by replacing the loop with a single array operation, which can significantly boost performance. 
+In scripting languages, **loops can be slow** to execute because of the overhead of the interpreter, which may need to parse each expression, perform various input data checks and more. These overheads add up when expressions are repeated many times in a loop. Vectorisation avoids the issue by replacing the loop with a single array operation. This can significantly boost performance if the array operation is implemented in a compiled language such as C. 
```

Comment:
Minor comment, but mentioning C might add some confusion around whether they actually need to write something in C and also how to tell whether an underlying operation is actually written in C. Also, it could be Fortran not C :-)

Reply:
Thanks @chrisdjscott . Mentioned C because we have a ctypes lesson for which I now added a link to (further down). There is no simple way to interface Python to Fortran except through f2py, which no longer looks like it is much developed. The problem with fortran is name mangling, different for each compiler. I think it's ok if we say "such as C"....